deepchem/models/torch_models/ferminet.py

```diff
@@ -22,9 +24,55 @@ def test_f(x: np.ndarray) -> np.ndarray:
     return 2 * np.log(np.random.uniform(low=0, high=1.0, size=np.shape(x)[0]))
 
 
-class Ferminet:
+class Ferminet(torch.nn.Module):
```

Comment:
Can you add this new layer to torch_layers.csv and layers.rst in the docs/?

Reply:
Can I do this once the model is fully completed? As I am breaking the chunk of PR into smaller ones and am adding one function by function.